sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/sampleoverview/SampleFilterImpl.java

```diff
@@ -22,25 +24,42 @@
  */
 public class SampleFilterImpl implements SampleFilter {
 
-    private final List<String> allowedStatuses = new ArrayList<>();
+  private final List<String> allowedStatuses = new ArrayList<>();
+  private String substring = "";
+
+  @Override
+  public SampleFilter withStatus(String status) {
+    allowedStatuses.clear();
+    allowedStatuses.add(status);
+    return this;
+  }
+
+  @Override
+  public SampleFilter containingText(String substring) {
+    this.substring = substring;
+    return this;
+  }
 
+  @Override
+  public Predicate<? extends Sample> asPredicate() {
+    final Predicate<Sample> containsText = it -> {
+      if (substring.isEmpty())
+        return true;
```

Comment:
Either put the if statement in brackets (I recall you taught me its how you should do it) or maybe remove it? You did not have this check in the project, I would prefer to have the same style for both of them :)

Reply:
good catch :+1: